cron_inactivity_period/i18n/fr.po

```diff
@@ -0,0 +1,95 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* cron_inactivity_period
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-08-22 10:52+0000\n"
+"PO-Revision-Date: 2018-08-22 10:52+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: cron_inactivity_period
+#: field:ir.cron.inactivity.period,inactivity_hour_begin:0
+msgid "Begin Hour"
+msgstr "Heure de début"
+
+#. module: cron_inactivity_period
+#: field:ir.cron.inactivity.period,create_uid:0
+msgid "Created by"
+msgstr "Created by"
+
+#. module: cron_inactivity_period
+#: field:ir.cron.inactivity.period,create_date:0
+msgid "Created on"
+msgstr "Created on"
+
+#. module: cron_inactivity_period
+#: field:ir.cron.inactivity.period,cron_id:0
+msgid "Cron id"
+msgstr "Cron id"
+
+#. module: cron_inactivity_period
+#: field:ir.cron.inactivity.period,display_name:0
+msgid "Display Name"
+msgstr "Display Name"
```

Comment:
Missing translation message

Reply:
Hi. display name and "last update on" ... are fields added automatically by Odoo in all new models. I don't think that adding translation is usefull. (I added however)